students/RimleeT/lesson02/series_fibonacci_lucas.py

```diff
@@ -0,0 +1,54 @@
+#Computing the Fibonacci and Lucas Series
+
+def fibonacci(n):
+	if n<2:
+		return n
+	return (fibonacci(n-2)+fibonacci(n-1))
+
+
+def lucas(n):
+	if n==0:
+		return 2
+	elif n==1:
+		return 1
+	else:
+		return (lucas(n-2)+lucas(n-1))
+
+"""
+sum_series: 
+It should have one required parameter and two optional parameters. The required parameter will determine which element in the series to print. The two optional parameters will have default values of 0 and 1 and will determine the first two values for the series to be produced.
+Calling this function with no optional parameters will produce numbers from the fibonacci series (because 0 and 1 are the defaults).
+Calling it with the optional arguments 2 and 1 will produce values from the lucas numbers.
+"""
+def sum_series(n, n0=0, n1=1):
+	if n0==2 and n1==1:
+		print("Calls lucas")
+		x=lucas(n)
+		return x
+	else:
```

Comment:
We don't want this to be always because our choices aren't always between lucas and fibonacci. You should be using generalized logic that can handle any input:

ex:
sum_series(3, 1, 1) = 3 (this needs to be implemented, it is very similar to the logic for fib/lucas)
    1, 1, 2, 3,
sum_series(3, 0, 1) = fib(3)
sum_series(3, 2, 1) = lucas(3)